xbmc/windowing/Resolution.cpp

```diff
@@ -20,6 +20,14 @@
 
 #include <cstdlib>
 
+EdgeInsets::EdgeInsets() : EdgeInsets(0.0f, 0.0f, 0.0f, 0.0f)
```

Comment:
Any reason for not using default member initialization?

Reply:
You mean using `= default`? Yes, good point! Thanks, will change.

Reply:
`float left = 0.0f;` in the struct definition is default member initialization. Once that is used you can then also use `= default`.